subsys/bluetooth/shell/bt.c

```diff
@@ -649,13 +662,10 @@ static int cmd_advertise(int argc, char *argv[])
 	param.interval_max = BT_GAP_ADV_FAST_INT_MAX_2;
 
 	if (!strcmp(argv[1], "on")) {
-		param.options = BT_LE_ADV_OPT_CONNECTABLE;
-		scan_rsp = sd;
-		scan_rsp_len = ARRAY_SIZE(sd);
+		param.options = BT_LE_ADV_OPT_CONNECTABLE |
+				BT_LE_ADV_OPT_USE_NAME;
```

Comment:
I'd put parenthesis around the bitwise operation. I think that's the general convention, checkpatch may complain otherwise, and it'll also make editors (at least mine) align the second line correctly :)

Reply:
Done